tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.Datadog.Trace.PublicApiHasNotChanged.verified.txt

```diff
@@ -125,6 +125,8 @@ namespace Datadog.Trace
         public const string ErrorMsg = "error.msg";
         public const string ErrorStack = "error.stack";
         public const string ErrorType = "error.type";
+        public const string GitCommitSha = "_dd.git.commit.sha";
```

Comment:
@andrewlock Is it correct to assume that this is fine, since all the other tags are public as well?

Reply:
Hah, I've commented earlier actually, I think all of these should be made `internal` actually. Especially `_dd.` tags; we generally _don't_ expect customers to be setting "internal implementation" tags like these.

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/commit/989c9c823ebb86da47f43eeaf48a2466a04e38af